freezegun/api.py

```diff
@@ -531,8 +531,8 @@ def __init__(self, time_to_freeze: datetime.datetime):
     def __call__(self) -> datetime.datetime:
         return self.time_to_freeze
 
-    def tick(self, delta: Union[datetime.timedelta, int]=datetime.timedelta(seconds=1)) -> datetime.datetime:
-        if isinstance(delta, numbers.Real):
+    def tick(self, delta: Union[datetime.timedelta, float]=datetime.timedelta(seconds=1)) -> datetime.datetime:
+        if isinstance(delta, float):
```

Comment:
It looks like this change causes the tests to fail.

If passing fractions causes this bit to fail, I think we should tackle that separately - this PR can just focus on the type fix.

Reply:
Sorry, my bad, it should be `isinstance(delta, (int, float))`. I got confused by `mypy` treating `int` as a subtype of `float`, but `isinstance` (correctly) doesn't.

As you suggested, I will split this off into another PR though. I theory, this could break people that managed to pass in some type that `isinstance` of `numbers.Real` *and* works with `datetime.timedelta(secods=x)`. Though, I couldn't come up with any such type other than `int` and `float`.